### DISCUSSION TITLE
Is there a planned date for the Scrubadub 2.0.0 Release ?

### DISCUSSION THREAD

**Discussion by mhsteven** [LeapBeyond/scrubadub#123] [General]
I like the Scrubadub 2.0.0 a lot, but I see it is in Pre-Release stage, is there a target date when we will have official 2.0.0 release to PyPI ?   Thanks!

**Comment by thomasbird** [comment]
No there is no release date planned. But, it will come in the next little while. Honestly, I was unhappy with how the extensibility was working in the pre release. I have a draft of how to make it work going forward which should be much more robust. So it'll be released in a little while.

**Reply by mhsteven** [reply]
Got it. Thank you Thomas ! I like that the pre-release now is able to integrate spacy for identify name which gives great accuracy. I look forward to its official release!

**Comment by thomasbird** [comment]
v2.0.0 has been released! Just FYI, the spacy part is now in a separate package: https://github.com/LeapBeyond/scrubadub_spacy You can checkout the new docs for more info: https://scrubadub.readthedocs.io/
